templates/security/cve/index.html

```diff
@@ -716,4 +716,4 @@ <h2>Recent CVEs affecting Ubuntu</h2>
 {% include "shared/contextual_footers/_contextual_footer.html" %}
 {% endwith %}
 
-{% endblock %}
+{% endblock %}
```

Comment:
This is probably because I copy pasted to avoid breaking changes in the conflicts when I was rebasing.

Reply:
Again just a newline character. Not a concern.